tests/components/litterrobot/test_select.py

```diff
@@ -63,3 +65,18 @@ async def test_invalid_wait_time_select(hass: HomeAssistant, mock_account):
             blocking=True,
         )
     assert not mock_account.robots[0].set_wait_time.called
+
+
+async def test_litter_robot_4_selects(
+    hass: HomeAssistant, mock_account_with_litterrobot_4: MagicMock
+) -> None:
+    """Tests Litter-Robot 4 selects."""
+    await setup_integration(hass, mock_account_with_litterrobot_4, PLATFORM_DOMAIN)
+
+    state = hass.states.get("select.test_night_light_level")
+    assert state.state == "Low"
+    assert state.attributes.get(ATTR_ICON) == "mdi:lightbulb-on-30"
```

Comment:
I'm not sure if we need to test the icons as they are aesthetic only. Perhaps we should should assert the possible options instead?

Reply:
Even though it's aesthetic, it's still code, so this ensures logic and coverage. But it is probably wise to add the options as a check as well.

Reply:
I like that answer and take back the first part. It's prone to errors so yeah. It seems best to keep the assertion.